compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/ClosureDesugar.java

```diff
@@ -1245,6 +1249,13 @@ public void visit(BLangRecordLiteral.BLangMapLiteral mapLiteral) {
 
     @Override
     public void visit(BLangRecordLiteral.BLangStructLiteral structLiteral) {
+        SymbolEnv symbolEnv = env.createClone();
+//        bLangLambdaFunction.capturedClosureEnv = symbolEnv;
+        BLangFunction enclInvokable = (BLangFunction) symbolEnv.enclInvokable;
+        // Save param closure map of the encl invokable.
+//        bLangLambdaFunction.paramMapSymbolsOfEnclInvokable = enclInvokable.paramClosureMap;
+//        boolean isWorker = bLangLambdaFunction.function.flagSet.contains(Flag.WORKER);
```

Comment:
commented out sections? do we need them?

Reply:
no, my bad, I though I removed before committing. will remove.